content/blog/2016/2016-08-03-docker-pipeline-environments.adoc

```diff
@@ -0,0 +1,120 @@
+---
+layout: post
+title: "Define your environment with Docker Pipeline, don't install software"
+tags:
+- jenkins
+- dsl
+- pipeline
+- plugins
+- blueocean
+- ux
+- javascript
+- nodejs
+author: michaelneale
+---
+
+If you are running parts of your pipeline on Linux, possibly the easiest way to get 
+a clean reusable environment is to use: link:https://go.cloudbees.com/docs/cloudbees-documentation/cje-user-guide/chapter-docker-workflow.html[Docker Pipeline].
+
+In this short post I wanted to show how you can avoid installing stuff on the agents, and have per project, or even per branch, customized build environments. 
+Your environment, as well as your pipeline is defined and versioned alongside your code. 
+
+This is used in the "dogfood" project for Blue Ocean. (see link:https://en.wikipedia.org/wiki/Eating_your_own_dog_food[eating own dogfood]). Some call it "drinking your own Champagne" but unless you live in France technically you can't say that.
```

Comment:
Why are you mentioning the "dogfood" project at this point?  This paragraph feels like it's missing something linking it to what comes before and after. 

Maybe "I'll use the "dogfood" project of Blue Ocean as an example"?